src/abstract.jl

```diff
@@ -97,6 +97,28 @@ The criterion refers to the *type* information, not the value. Hence, e.g. a
 """
 function ispolynomial(::AbstractSystem) end
 
+"""
+    isnoisy(s::AbstractSystem)
+
+Specifies if the dynamics of system `s` contains a noise term `w`.
```

Comment:
```suggestion
Determines if the dynamics of system `s` contains a noise term `w`.
```
(same for the other docstrings)